footer.md

```diff
@@ -1,35 +1,40 @@
-<h1></h1>
```

Comment:
Doesn't removing this again make us end up with a bigger title?

Reply:
OK, just curious how we replicate what we had before: a smaller header.

Reply:
For now, you can add a style rule to the `.footer` block:
```css
  & h1 {
    font-size: 1.5rem;
  }
```